src/fleet/azext_fleet/custom.py

```diff
@@ -212,6 +212,97 @@ def get_credentials(cmd,  # pylint: disable=unused-argument
         raise CLIError("Fail to find kubeconfig file.")
 
 
+def assign_fleet_identity(cmd,
```

Comment:
I think there is significant trouble here.  Let's discuss on a call.

Reply:
Discussed, closing PR as discussed. Tabling az identity sub command changes.